lib/models/repositories.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:hive/hive.dart';
+
+import '../exports.dart';
+
+part 'repositories.g.dart';
+
+@HiveType(typeId: 1, adapterName: 'UserAuthenAdapter')
+class UserAuthen extends HiveObject {
```

Comment:
Could we use `AuthFactor`? Also, we might implement "Remember username" by default.